src/Poseidon/Utils.hs

```diff
@@ -1,7 +1,7 @@
 {-# LANGUAGE OverloadedStrings #-}
 
 module Poseidon.Utils (
-    PoseidonException(..),
+    PoseidonException (..),
```

Comment:
This clashes with my coding standard, dear Sir. We need to call a meeting.

Reply:
Hahaha - ok - maybe I also need one of these neat linters.